### PR TITLE
Stop builds also for closed PRs that were not merged

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
I don't know why we had the extra check before for not stopping build on closed PRs. It didn't make much sense to me.

The GitHub documentation can be found here: https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/webhook-events-and-payloads#pull_request

File: main.go [hunk 1]
```diff
@@ -1073,31 +1073,30 @@ func getBuildParameters(conf *config, build *buildOptions) ([]*gitlab.PipelineVa
...
+func stopBuildsOfStalePRs(pr *github.PullRequestEvent, conf *config) error {
 
-	// If the action is "closed" and the "merged" key is "true", the pull request was merged.
-	// While webhooks are also triggered when a pull request is synchronized, Events API timelines
-	// don't include pull request events with the "synchronize" action.
-	if !(pr.GetAction() == "closed" && pr.PullRequest.GetMerged()) {
-		log.Debugf("stopBuildsOfMergedPRs: PR not merged and closed")
+	// If the action is "closed" the pull request was merged or just closed,
+	// stop builds in both cases.
+	if !(pr.GetAction() == "closed") {
```
Comment (oleorhagen) [inline main.go:1081]:
```suggestion
	if pr.GetAction() != "closed" {
```

File: main.go [hunk 2]
```diff
@@ -1073,31 +1073,30 @@ func getBuildParameters(conf *config, build *buildOptions) ([]*gitlab.PipelineVa
...
+		log.Debugf("stopBuildsOfStalePRs: PR not merged and closed")
```
Comment (oleorhagen) [inline main.go:1082]: Maybe not the correct log message anylonger, how about:
```suggestion
		log.Debugf("stopBuildsOfStalePRs: PR not closed, therefore not stopping it's pipeline")
```
Or something along these lines.

**Review by oleorhagen — APPROVED** [review]
Should do the trick

**Comment by oleorhagen** [comment]
> I don't know why we had the extra check before for not stopping build on closed PRs. It didn't make much sense to me.
> 
I guess simply cause we didn't think about it the last time around:

https://tracker.mender.io/browse/QA-122

**Pull request merged**